src/connection/runtime.rs

```diff
@@ -44,7 +44,7 @@ impl BackgroundRuntime {
         thread::Builder::new().name("gRPC worker".to_string()).spawn(move || {
             async_runtime.block_on(async move {
                 shutdown_source.recv().await;
-            })
+            });
```

Comment:
Not strictly necessary, but more consistent. Adding a semicolon here makes it explicit that the closure returns `()`.